AteaTask1.Core/Interfaces/ITableStorageService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Azure;
+using AteaTask1.Core.Models;
+
+namespace AteaTask1.Core.Interfaces
+{
+    public interface ITableStorageService
+    {
+        Task AddRecordAsync(string key, bool isSuccess);
+        Pageable<Record> GetLogsFromTo(DateTime from, DateTime to);
```

Comment:
GetLogsAsync? 
nezinu vai from to ir kāds added value ja tie jau ir argumenti.. :? 
from un to var pārsaukt par periodStart un periodEnd... tas varbūt palīdz to FromTo aizstāt?